ruby_training/ruby_classes/csv.rb

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+p CSV.read("data.csv")
+
+p CSV.parse("1,one\n2,two\n3,three")
+
+table = CSV.parse(File.read("data.csv"),headers: true)
+p table[2]["name"]
+
+p table.by_col[0]
+p table.by_col[1]
+
+p CSV.parse("1,2,3",converters: :numeric)
+
+#=====================================================================
+# creating csv files
+
+names = [[1,"ali"],[2,"ahmad"],[3,"osama"]]
+p names.map { |a| a.join(",")}.join("\n")
```

Comment:
what if you want to write in csv file?

Reply:
open CSV file.
file = CSV.open("filename")
file<<[ "a", "b", "c", "d" ]
file.close()